lib/routes/substack/subscribe.ts

```diff
@@ -29,7 +30,8 @@ async function handler(ctx) {
         throw new InvalidParameterError('Invalid user');
     }
 
-    const feed = await parser.parseURL(`https://${user}.substack.com/feed`);
+    const response = await ofetch(`https://${user}.substack.com/feed`);
```

Comment:
Please add a check for `user` using https://github.com/DIYgod/RSSHub/blob/b894551e4e85fd4808c7d11a0671224314dd473a/lib/utils/valid-host.ts#L6 before requesting